README.md

```diff
@@ -130,6 +130,13 @@ machine being provisioned, which may be a security concern.
 
 An optional array of cookies to add to the HTTP request for the download.
 
+#### `allow_insecure_ssl`
+
+Permits the use of sslv3 HTTPS connections. If this is set to false, only TLSv1, TLSv1.1
+and TLSv1.2 are permitted.
+
+Defaults to true. 
```

Comment:
is it really a good idea that it defaults to true? Is is pretty insecure and our defaults should be as secure as possible.

Reply:
@bastelfreak Initially the PR disabled sslv3. @ofalk raised concerns about disabling sslv3 potentially being a breaking change for a lot of sites. If Puppet has a standard here, I'm happy to update the PR to reflect that.

Reply:
okay valid point. Let's stick with the current settings and change it somewhere in a future release

Reply:
I think the module should probably default to the secure setting though.  This would be a breaking change for the module, but I think that's OK as long as we mark it as such and have documented the new flag.  Also, isn't TLS 1.0 also considered insecure these days?